src/lib.rs

```diff
@@ -87,4 +93,10 @@ impl RobloxStudio {
     pub fn built_in_plugins_path(&self) -> PathBuf {
         self.root.join("BuiltInPlugins")
     }
+
+    #[must_use]
+    #[inline]
+    pub fn plugins_path(&self) -> PathBuf {
```

Comment:
I think I'd rather want this as `&Path`, and if that's not possible to be a `Cow`.